service/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:8-jre
+
+COPY geotrellis/target/scala-2.11/geotrellis-chatta-demo-assembly-0.1-SNAPSHOT.jar /opt/
+COPY geotrellis/data/chatta-demo /data/chatta-demo
+COPY static/ /static
+
+EXPOSE 8777
+
+CMD java -cp /opt/geotrellis-chatta-demo-assembly-0.1-SNAPSHOT.jar geotrellis.chatta.Main
```

Comment:
I think that it would be better to convert this to a `ENTRYPOINT` and `CMD` combo vs. just `CMD`.